### PR TITLE
feat(Team Dashboard): Temporarily hide team stats block

### PR DESCRIPTION
**Pull request by HenerHoop**
https://app.clickup.com/t/86ayqffyf

### CODE REVIEW

File: packages/cxl-ui/src/components/cxl-dashboard-team-stats.js [hunk 1]
```diff
@@ -16,6 +16,10 @@ export class CxlDashboardTeamStatsElement extends LitElement {
 
   @property({ type: String, attribute: 'manage-roadmaps-url' }) manageRoadmapsURL = '';
 
+  @property({ type: Boolean, attribute: 'show-progress' }) showProgress = false;
+
+  @property({ type: Boolean, attribute: 'show-stats' }) showStats = false;
+
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-dashboard-team-stats.js:22]: I suggest to make it `true` by default as it's the most desired situation. Only in some cases we'd like to hide it.

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

File: packages/cxl-ui/src/components/cxl-dashboard-team-stats.js (hunk 1 above)
Comment (HenerHoop) [reply packages/cxl-ui/src/components/cxl-dashboard-team-stats.js]: I flipped the logic to make it a bit clearer. Instead of show, it's now hide.

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Commit message

```diff
- feat(Team Dashboard): Temporarily hide team stats block
+ feat(cxl-ui): cxl-dashboard-team-stats add hide progress, stats properties
```

---

> I flipped the logic to make it a bit clearer. Instead of show, it's now hide.

IMO using
`${this.showProgress ? html`` : nothing`

is easier to read than
`${!this.hideProgress ? html`` : nothing`

but let it stay and not waste time on opinions ;)

**Comment by HenerHoop** [comment]
> Commit message
> 
> ```diff
> - feat(Team Dashboard): Temporarily hide team stats block
> + feat(cxl-ui): cxl-dashboard-team-stats add hide progress, stats properties
> ```
> 
> > I flipped the logic to make it a bit clearer. Instead of show, it's now hide.
> 
> IMO using `` ${this.showProgress ? html`` : nothing ``
> 
> is easier to read than `` ${!this.hideProgress ? html`` : nothing ``
> 
> but let it stay and not waste time on opinions ;)

I agree with you. From a developer's perspective, it might be a bit harder to read, but in terms of overall logic, it seems more logical to me that the element is visible in normal situations, and then we want to **hide** it in certain cases.

**Review by pawelkmpt — APPROVED** [review]

**Comment by pawelkmpt** [comment]
@HenerHoop conflicts appeared after merging #361. Please resolve them

**Pull request merged**